Dockerfile

```diff
@@ -1,8 +1,19 @@
-FROM ubuntu:latest
-
-EXPOSE 8555
-EXPOSE 8444
+FROM python:slim AS builder
+ARG BRANCH="main"
+RUN DEBIAN_FRONTEND=noninteractive apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y git build-essential
+RUN python -m venv /chia-blockchain/venv
+ENV PATH /chia-blockchain/venv/bin:$PATH
+WORKDIR /chia-blockchain/src
+RUN git clone --branch $BRANCH https://github.com/Chia-Network/chia-blockchain.git .
+RUN git submodule update --init mozilla-ca
+RUN pip install wheel
+RUN pip install --extra-index-url https://pypi.chia.net/simple miniupnpc==2.1
+RUN pip install --extra-index-url https://pypi.chia.net/simple .
```

Comment:
I get why this is simpler than calling install.sh like we do now, though I'm still leaning towards calling install.sh anyways, to make maintenance easier in the long run, especially since this layer gets thrown away anyways - its much easier to automatically inherit changes to install.sh that might happen upstream in the chia-blockchain repo than to have to remember to change it in both places if it ever changes